app/main/models/users.py

```diff
@@ -52,14 +54,82 @@ class User(db.Model):
 	creation_time = db.Column(db.DateTime)
 
 	#Relationships
+	tags = db.relationship('Tag', secondary=userTagJunction, lazy='subquery',
+							backref=db.backref("users", lazy=True))
+
+	saves = db.relationship('Post', secondary=userPostInteraction, lazy=True,
+							backref=db.backref("savers", lazy=True))
+
+	#To get all payments done by user, call User.payments
+	#This is defined in payments.py as db.relationship
+
+	def __init__(self, username, password, email):
+		self.username = username
+		self.password = password
+		self.email = email
+
+		db.session.add(self)
+		db.session.commit()
+
+	def resetPassword(self, newPassword):
+		#Pass in a hashed password
+		self.password = newPassword
+		db.session.commit()
+
+	def updateMetaData(self, **kwargs):
+		# TODO
+		pass
+	def getMetaData(self):
+		# TODO
+		pass
+
+	# TODO: Do something about isVerifiedEmail 
```

Comment:
Add a database column that will store whether email is verified or not, and from here give `isVerified` and `setVerified` methods.